spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnRef.scala

```diff
@@ -17,6 +17,9 @@ sealed trait NamedColumnRef extends ColumnRef {
   /** Returns a name of the selection as it is seen in the result set. Most likely this is going to be
     * used when providing custom column name to field name mapping. */
   def selectedAs: String
+
+  /** Returns an alias for the referenced column */
+  def alias: Option[String]
```

Comment:
This is an `Option` because alias is optional. And if it is not used, something else is tried - look at my above comment related to `AllColumns`.